lib/setupmanagers/qemuhck/network_manager.rb

```diff
@@ -38,6 +38,19 @@ def read_device(device)
         Json.read_json(device_json, @logger)
       end
 
+      def read_world_ip(device_name, bridge_name, qemu_replacement_list = {})
+        device = read_device(device_name)
+        type_config = @config['devices']['world']
+        replacement_list = device_replacement_list('world', device, type_config, qemu_replacement_list)
+        mac = replace_string_recursive(type_config['mac'], replacement_list)
+
+        line = File.readlines('/proc/net/arp')[1..].map(&:split).find do
```

Comment:
The idea looks good, but I am not sure about the stability. When I ran VM and then stopped it (without network name space), VM MAC still was present in the arp table for a long time.

Reply:
It shouldn't be a problem since the ARP table is isolated by network namespace.